Views/StatisticsScreen.xaml

```diff
@@ -24,13 +28,12 @@ Statistics screen -->
                 </StackLayout>
 
                 <!-- Box plot chart -->
-                <StackLayout Padding="10">
-                    <Image Source="https://thirdspacelearning.com/wp-content/uploads/2023/07/Box-Plot-image-40-US.png"
-                           HorizontalOptions="Center"
-                           VerticalOptions="CenterAndExpand"
-                           WidthRequest="300"
-                           Margin="0,0,0,0"/>
-                </StackLayout>
+                <GraphicsView 
+                x:Name="Canvas"                 
+                HorizontalOptions="Fill"
+                Drawable="{StaticResource MyDrawable}" 
+                HeightRequest="100"       
+                Grid.Row="1"/>
 
                 <!-- Stats section -->
                 <StackLayout BackgroundColor="#E6E6E6"
```

Comment:
I was able to get the statistics page to load and everything looked great except I think the text for the stats might be a little bit small. I think it will look better if the font is a little bigger, so it might be worth testing a couple different sizes and picking which ever one looks best.

Reply:
Thanks for the suggestion! I just made that change :)